egs2/mini_an4/asr1/run_adapters.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+#run this shell file to test out adapter aided training.
+
+set -e
+set -u
+set -o pipefail
+
+./asr.sh \
+    --lang en \
+    --asr_config conf/adapter_example.yaml \
```

Comment:
Did you forget to add this config?

Reply:
Hi, yes it is correct that none of the modules would be called. They would not be able to be used because the function that calls them will go in #5035. The reason why the functions that calls them is _not there_ is because should I add them in, they would not pass CI -- CI would say I am importing `adapters` which does not exist yet.

Reply:
Thanks! #5035 seems to have only two adapter files. How about including those two files in this PR? Please remind me if there is any error.

Reply:
Yes, I will update the rest of the files later today to #5035

Reply:
all changes completed

Reply:
included the updated version those two files